src/app/components/progress-indicator/passive-progress-indicator/index.html

```diff
@@ -1,287 +1,3 @@
-        </sky-demo-page-property>
-        <sky-demo-page-property
-          propertyName="disabled"
-          isOptional="true"
-          defaultValue="false"
-        >
-          Indicates whether to disable the reset button. This property accepts <sky-code>boolean</sky-code> values.
-        </sky-demo-page-property>
-      </sky-demo-page-properties>
-
-      <sky-demo-page-properties sectionHeading="Progress indicator reset button events">
-        <sky-demo-page-property
-          propertyName="resetClick"
-          isOptional="true"
-        >
-          Fires when users select the reset button that marks all items as incomplete and sets the first progress item as the active item.
-        </sky-demo-page-property>
-      </sky-demo-page-properties>
-
-      <sky-demo-page-properties sectionHeading="Progress indicator nav button properties">
-        <p>
-          The progress indicator nav button component allows users to navigate the steps in modal wizards. We recommend against using it in waterfall pages and passive progress indicators.
-        </p>
-        <sky-demo-page-property
-          propertyName="progressIndicator"
-        >
-          Specifies the progress indicator component to associate with the nav button. This property accepts an array or Observable array of <sky-code>string</sky-code> values.
-        </sky-demo-page-property>
-        <sky-demo-page-property
-          propertyName="buttonText"
-          isOptional="true"
-          defaultValue="Next"
-        >
-          Specifies the label to display on the nav button. The property accepts <sky-code>string</sky-code> values.
-        </sky-demo-page-property>
-        <sky-demo-page-property
-          propertyName="buttonType"
-          isOptional="true"
-          defaultValue="next"
-        >
-          Specifies the type of nav button to include. The valid options are <sky-code>finish</sky-code>, <sky-code>next</sky-code>, and <sky-code>previous</sky-code>. This property accepts <sky-code>string</sky-code> values.
-        </sky-demo-page-property>
-        <sky-demo-page-property
-          propertyName="disabled"
-          isOptional="true"
-          defaultValue="false"
-        >
-          Indicates whether to disable the nav button. This property accepts <sky-code>boolean</sky-code> values.
-        </sky-demo-page-property>
-      </sky-demo-page-properties>
-
-      <sky-demo-page-example>
-        <sky-progress-indicator-passive-demo>
-        </sky-progress-indicator-passive-demo>
-        <sky-demo-page-code demoName="Progress indicator — passive"></sky-demo-page-code>
-      </sky-demo-page-example>
-    </sky-tab>
-  </sky-tabset>
-
-</sky-tabbed-demo-page>
+<sky-docs-url-redirect
+  url="https://developer.blackbaud.com/skyux-progress-indicator/docs/waterfall-indicator"
```

Comment:
@Blackbaud-DeniseCisneros why are we redirecting to the waterfall indicator instead of the passive indicator?

Reply:
Ah, poor copy/paste error on my end. Great catch! Will fix that now